doc/api/esm.md

```diff
@@ -83,12 +83,19 @@ All CommonJS, JSON, and C++ modules can be used with `import`.
 Modules loaded this way will only be loaded once, even if their query
 or fragment string differs between `import` statements.
 
-When loaded via `import` these modules will provide a single `default` export
-representing the value of `module.exports` at the time they finished evaluating.
+CommonJS modules, when imported, will be handled in one of two ways. By default
+they will provide a single `default` export representing the value of
+`module.exports` at the time they finish evaluating. However, they may also
+provide `@@esModuleInterop` or `__esModule` to use named exports, representing
```

Comment:
Perhaps clarify exactly what these properties are - "provide a `@esModuleInterop` or `__esModule` boolean export indicating that a snapshot should be taken of each enumerable key..."

Reply:
sounds good, and is it `@@` or `@` to doc a symbol, i thought it was two

Reply:
Yeah it's still two, that was a typo.